examples/webpack.config.js

```diff
@@ -122,17 +124,8 @@ module.exports = async (env) => {
                 uglifyOptions,
                 parallel: true
             }),
-            new CompressionPlugin({
-                asset: '[file].gz',
```

Comment:
this configuration was moved ? or its default config and its not needed anymore?

Reply:
Yeah this is default config.